src/Controls/src/Core/Region.cs

```diff
@@ -26,6 +27,12 @@ public struct Region : IEquatable<Region>
 			_inflation = inflation;
 		}
 
+		internal static Region FromRectangles(IEnumerable<Rect> rectangles)
```

Comment:
Should this rather be a new ctor that takes IEnumerable? Is there a reason for the list-only approach?

Reply:
Yes, it would be possible but it would have to be public and would require changes to the public APIs of all supported platforms.

Reply:
Maybe this should be public for main/net8 and internal for net7 since we can no longer use the lines to work?